pkg/geo/wkt/wkt.y

```diff
@@ -42,31 +44,38 @@ start:
 
 geometry:
 	point
+|	linestring
+	{
+		// Check that non-empty linestrings have at least 2 points.
```

Comment:
@otan Should I take this out?

Reply:
yeah i think so, especially if we're not returning a more descriptive error message here.